SMLHelper/Assets/ModPrefab.cs

```diff
@@ -62,6 +65,14 @@ protected ModPrefab(string classId, string prefabFileName, TechType techType = T
 
         internal GameObject GetGameObjectInternal()
         {
+            if(cachedPrefab)
+            {
+                // This will avoid GetGameObject() being called every time the object is spawned. Could break things if there is a degree 
+                // of random generation in GetGameObject(), in that case the modder will need to do their random stuff in a MonoBehaviour.Awake()
+                // e.g. in MinecraftFish, the tropical fish have random colours which is done in a MonoBehaviour, rather than the GetGameObject() method
+                return cachedPrefab;
```

Comment:
After some more review, I am suggesting that we make this an optional feature.  
Any mods that conditionally return `null` from `GetGameObject` (intentionally) will break with this change.

I have a couple of mods that do that today and it stands to reason that other mods, either today or in the future, may do the same.
https://github.com/PrimeSonic/PrimeSonicSubnauticaMods/blob/master/CyclopsNuclearReactor/CyNukReactorBuildable.cs#L108
https://github.com/PrimeSonic/PrimeSonicSubnauticaMods/blob/master/CyclopsBioReactor/Items/CyBioReactor.cs#L69

If caching were made optional and default it to "do not cache" to preserve compatibility with the behavior of existing mods,  
I would expect that the feature would almost never be used.

Considering how little impact comes from this one method, I suggest that the sane thing to do is to remove prefab caching entirely.

Reply:
My opinion is that prefab caching should be removed.

Reply:
For static objects that cached prefab is deleting in Fixer component after 1 second anyway.  It now works only for physics enabled objects and only because prefabs are sinking down infinitely.

Reply:
So Buildables aren't being removed by the Fixer component.  
That is something that will need to be resolved separately.